### DISCUSSION TITLE
Another weird issue with raycasts and extern systems (this time with VSync enabled)

### DISCUSSION THREAD

**Discussion by frutas-fruit** [xissburg/edyn#106] [General]

**Comment by xissburg** [comment]
Oh yes you're gonna have problems if a component is being modified in both registries as it is constantly trying to override its local value with the remote value. Generally, only one registry should own the value of a component and it is read-only in others. It looks like you have to split your component into input values and calculated values. The input values are set by the user in the main registry and then it is propagated to the islands and external systems. The calculated or procedural values should be in another component which is modified in the external system and then overridden into the main thread later by marking it as dirty.

I still need to test raycasting in external systems to see what is going on with the other issue you have...
